src/app/components/administrator/dashboard/add-users/add-judges/add-judges.component.html

```diff
@@ -0,0 +1,56 @@
+<div class="flex-container">
+    <div class="top-options">
+        <h1>Agregar Juez</h1>
+        <h3>Crea una cuenta para un nuevo juez</h3>
+      </div>
+      <form class="add-form">
```

Comment:
You should use flexbox to separate the boxes of the inputs evenly. You can use as an example the create project component, there is a form structured to separate the boxes and with a better look and feel.